README.md

```diff
@@ -78,6 +78,35 @@ Currently you must use a version of TensorFlow from
 instructions](#build-the-tensorflow-backend-with-custom-tensorflow)
 below.
 
+### How does the TensorFlow backend manage GPU memory?
+
+The TensorFlow backend does not "release" GPU memory until the Triton process
+exits. TensorFlow uses a pool allocator and so it retains any memory it
+allocates until its own process exits. It will reuse that memory if you load
+another TensorFlow model, but it will not return it to the system, even if it
+is no longer using it. For this reason, it is preferred to keep TensorFlow
+models grouped together on the same Triton process if you will be repeatedly
+loading/unloading them.
+
+From the TensorFlow GPU docs: "[Memory is not released since it can lead to memory fragmentation](https://www.tensorflow.org/guide/gpu#limiting_gpu_memory_growth)".
+
+#### Workarounds
```

Comment:
Following are few available options to limit the total amount of memory that TensorFlow allocates: